### PR TITLE
Update 'Install kubeadm/kubectl' page to turn off YUM's `repo_gpgcheck`

### LINKED ISSUE #32207: Issue with the gpgkeys for Kubernetes repository
_[When trying to follow advice in the page]_ Install and Set Up kubectl on Linux - [Install using native package management](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/#install-using-native-package-management), Red Hat-based distributions:

we are facing an issue when installing kubectl on Redhat/centos7 VM, and when applying:

1. `yum repolist`
```
kubernetes/signature                                                                                                                                                                                                  |  844 B  00:00:00
Retrieving key from https://packages.cloud.google.com/yum/doc/yum-key.gpg
Importing GPG key 0x7AA99B80:
 Userid     : "Rapture Automatic Signing Key (cloud-rapture-signing-key-2022-03-07-08_01_01.pub)"
 Fingerprint: e89b faec 34d3 1829 362f 7f7b 5668 44f3 7aa9 9b80
 From       : https://packages.cloud.google.com/yum/doc/yum-key.gpg
```
2. `cat /etc/yum.repos.d/kubernetes.repo`
```
[kubernetes]
baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-x86_64
gpgcheck=True
repo_gpgcheck=True
gpgkey=https://packages.cloud.google.com/yum/doc/yum-key.gpg
                 https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
```

### PR DESCRIPTION
**Pull request by jihoon-seo**
This PR closes #32207.
CC @sftim @BenTheElder (from #32207)

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ name=Kubernetes
 baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch
 enabled=1
 gpgcheck=1
-repo_gpgcheck=1
+repo_gpgcheck=0
```
Comment (BenTheElder) [inline content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:185]: I think we can just delete this line, and let the default of 0 take effect, which also has less appearance of endorsing disabling it. https://linux.die.net/man/5/yum.conf

Comment (jihoon-seo) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md:185]: @BenTheElder I checked using `centos:7` Docker image, and confirm that it works with this line deleted.
I will reflect your suggestion. 😊

Comment (jihoon-seo) [reply content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: Done! PTAL again 😊

**Review by BenTheElder — APPROVED** [review]
/lgtm
thank you!

**Review by sftim — COMMENTED** [review]
Let's do this.

/approve

**Pull request merged**